querydsl-maven-plugin/src/main/java/com/querydsl/maven/AbstractMetaDataExportMojo.java

```diff
@@ -394,18 +395,10 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             }
 
             MetaDataExporter exporter = new MetaDataExporter();
-            if (namePrefix != null) {
-                exporter.setNamePrefix(namePrefix);
-            }
-            if (nameSuffix != null) {
-                exporter.setNameSuffix(nameSuffix);
-            }
-            if (beanPrefix != null) {
-                exporter.setBeanPrefix(beanPrefix);
-            }
-            if (beanSuffix != null) {
-                exporter.setBeanSuffix(beanSuffix);
-            }
+            exporter.setNamePrefix(emptyIfSetToBlank(namePrefix));
+            exporter.setNameSuffix(Strings.nullToEmpty(nameSuffix));
+            exporter.setBeanPrefix(Strings.nullToEmpty(beanPrefix));
+            exporter.setBeanSuffix(Strings.nullToEmpty(beanSuffix));
```

Comment:
I figured changing it also for these is not necessary, since only `namePrefix` has a `default-value` set to a non-empty String.